PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
-Please make sure you have done the following:
+# Please make sure you have done the following:
 
-- [ ] Created a folder with the name of language you code inside `/Day<n>/` directory (Where n is for nth day)
-- [ ] Added your Codes
+- [ ] Created a folder with the name of language you code inside directory for the day you are making the contribution to
```

Comment:
Seems like the earlier version was clearer.

Reply:
This is perfect

Thanks a lot @aaditkamat 

@SvA1 please make these changes